source/references/components/refresh.md

```diff
@@ -1,222 +1,141 @@
+ - Simple example :
 
- ```
-  dy: the differencen between two scroll actions
-  pullingDistance: the distance of pulling
-  viewHeight: the height of refreshView
-  type: "pullingdown" constant string type for this event
- ```
-
-### Restrictions
-
-* refresh/loading does not support remove action, may support in Weex 0.9.
-* refresh/loading despite setting with display='hide', the refresh/loading view will still appear when scrolling due to known issues. it can be fixed with a another display='hide' when the refresh/loading should be hidden.
-* refresh/loading can only be hidden or displayed with an attribute display with value of show or hide. And there should be a statement of display='hide' when display='show' shows up in an event function, or your scroller may not response to user inputs.
-
-### example
-
-```html
-<template>
-  <scroller class="scroller">
-    <refresh class="refresh" @refresh="onrefresh" @pullingdown="onpullingdown" :display="refreshing ? 'show' : 'hide'">
-      <text class="indicator">Refreshing ...</text>
-    </refresh>
-    <div class="cell" v-for="num in lists">
-      <div class="panel">
-        <text class="text">{{num}}</text>
-      </div>
-    </div>
-  </scroller>
-</template>
+```
+<scroller>
+  <refresh @refresh="onrefresh" @pullingdown="onpullingdown">
+    ...
+  </refresh>
+  ...
+</scroller>
 
 <script>
-  const modal = weex.requireModule('modal')
-
   export default {
-    data () {
-      return {
-        refreshing: false,
-        lists: [1, 2, 3, 4, 5]
-      }
-    },
     methods: {
       onrefresh (event) {
-        console.log('is refreshing')
-        modal.toast({ message: 'refresh', duration: 1 })
-        this.refreshing = true
-        setTimeout(() => {
-          this.refreshing = false
-        }, 2000)
+        ...
       },
       onpullingdown (event) {
-        console.log('is onpulling down')
-        modal.toast({ message: 'pulling down', duration: 1 })
+        console.log("dy: " + event.dy)
+        console.log("pullingDistance: " + event.pullingDistance)
+        console.log("viewHeight: " + event.viewHeight)
+        console.log("type: " + type)
       }
     }
   }
 </script>
-
-<style scoped>
-  .indicator {
-    color: #888888;
-    font-size: 42px;
-    text-align: center;
-  }
-  .panel {
-    width: 600px;
-    height: 250px;
-    margin-left: 75px;
-    margin-top: 35px;
-    margin-bottom: 35px;
-    flex-direction: column;
-    justify-content: center;
-    border-width: 2px;
-    border-style: solid;
-    border-color: #DDDDDD;
-    background-color: #F5F5F5;
-  }
-  .text {
-    font-size: 50px;
-    text-align: center;
-    color: #41B883;
-  }
-</style>
 ```
+ - Complete example goes [here](http://dotwe.org/vue/26937c1c74022e79608af118b21bfbc7)
+
+## Usage Notes
+
+ - On the Android platform, it is not allowed for the refresh view to be pushed back when it is been pulling down or its onrefresh event is tiggered, while it is allow as for iOS platform and HTML5.
```

Comment:
There is a restriction on Android platform that the refresh view cannot be pushed back during the pulling down process or the`onrefresh` event is triggered, while it is allowed both on iOS and HTML5.